manifests/operator.yaml

```diff
@@ -83,4 +83,8 @@ spec:
           env:
             - name: OPERATOR_NAME
               value: "m3db-operator"
+            - name: ENVIRONMENT
+              value: "production" 
+            - name: KUBE_CFG_FILE
```

Comment:
Do we need to specify this (even if it's empty) when we're running in-cluster?

Reply:
I figured that it would be a place holder for the human operator to know that is an option? Like documenting various configurations?